src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (int i = 0; i < records.length; i++) {
+            String[] yourVariable = records[i].split(":");
```

Comment:
```suggestion
            String[] record = records[i].split(":");
```
your variable that was just example)